util-cache/src/main/scala/sbt/util/FileInfo.scala

```diff
@@ -115,6 +115,8 @@ object FileInfo {
     }
 
     implicit def apply(file: File): HashFileInfo = FileHash(file.getAbsoluteFile, computeHash(file))
+    implicit def apply(file: File, bytes: List[Byte]): HashFileInfo =
```

Comment:
Is there a reason to keep `implicit` here? Maybe it's better to distinguish the two `apply`s using something like `of`?

Reply:
No. Good catch.

Reply:
I don't really think it's necessary to disambiguate with `of` vs `apply`. The change is more or less the same as adding default parameters to the implicit apply (though you can't actually do this with default parameters since the defaults would depend on the file parameter).